core/src/main/java/org/apache/druid/indexer/TaskStatusPlus.java

```diff
@@ -226,6 +241,7 @@ public String toString()
   {
     return "TaskStatusPlus{" +
            "id='" + id + '\'' +
+           "groupId='" + groupId + '\'' +
```

Comment:
should this have a `, ` to match the rest of the pattern?

Reply:
oops, yes, i'll give you that comma !